cats/models.py

```diff
@@ -0,0 +1,99 @@
+"""
+Models used in app
+"""
+import datetime
+
+from django.core.exceptions import ValidationError
+from django.db import models
+
+STATUS_CHOICES = [
+    ('0', 'No status'),
+    ('1', 'Pending'),
+    ('2', 'Actual'),
+    ('3', 'Finished'),
+    ('4', 'Cancelled'),
+]
+
+
+class TimeStamped(models.Model):
+    """TimeStamp using when creating new object"""
+    created = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
+
+
+class CatQuerySet(models.QuerySet):
+    def filter_by_dates(self, rental_date, return_date):
```

Comment:
maybe better name would be `get_available_cats`

Reply:
True, this name wasn't saying much about result of the method